InvenTree/InvenTree/tasks.py

```diff
@@ -306,4 +307,35 @@ def send_email(subject, body, recipients, from_email=None):
         from_email,
         recipients,
         fail_silently=False,
+        html_message=html_message
     )
+
+
+def notify_low_stock(stock_item):
```

Comment:
You should pass a `Part` object to the function, not a `StockItem`.

The reason is that the *Part* is what we are concerned about running low on stock.

Say you have a capacitor with a low stock threshold of 2,000 units. You have the following stock items:

- A) One reel of 4,000
- B) One reel of 4,000
- C) One reel of 2,500

You remove 600 units from C) which brings it down to 1,900 units. This individual stock item is below the threshold, but there is still sufficient stock of the Part.

The "low stock threshold" should only be activated when the **total stock** for a given *Part* dips below the low stock threshold.